web/client/src/routes/+page.svelte

```diff
@@ -17,15 +17,24 @@
   import { quintOut } from 'svelte/easing';
 
   /**
- * @typedef CreateJobParameters
- * @type {object}
- * @property {string=} url - job source url.
- * @property {string} countryCode - country code.
- */
+   * @typedef CreateJobParameters
+   * @type {object}
+   * @property {string=} url - job source url.
+   * @property {string} countryCode - country code.
+   */
+
+  /**
+   * @typedef { 'authorizing'
+   *  | 'uploading'
+   *  | 'processing'
+   *  | 'ready'
+   * }
+   * Status
+   */
 
   let showDocs = true;
 
-  const apiRoot = 'https://gtfs-validator-web-mbzoxaljzq-ue.a.run.app'
+  const apiRoot = 'https://gtfs-validator-web-mbzoxaljzq-ue.a.run.app';
```

Comment:
[out of scope]: Not part of this PR, we should make the API endpoint a parameter in the CI process. This will allow in the future to deploy the same UI code in different environments(Staging vs Production).

Reply:
Also, after discussing with @themightychris , it may make sense to use a developer config for the web validator client so that local development doesn't inadvertently log errors to the production instance. 

When we implement the CI process changes, I will be sure to have two environments configured to avoid this scenario.